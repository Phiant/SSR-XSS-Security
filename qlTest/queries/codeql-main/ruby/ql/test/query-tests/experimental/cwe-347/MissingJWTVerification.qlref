experimental/cwe-347/MissingJWTVerification.ql