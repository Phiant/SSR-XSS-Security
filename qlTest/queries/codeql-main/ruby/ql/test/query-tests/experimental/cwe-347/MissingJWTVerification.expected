| MissingJWTVerification.rb:12:29:12:51 | token_without_signature | is not verified with a cryptographic secret or public key. |
| MissingJWTVerification.rb:15:29:15:51 | token_without_signature | is not verified with a cryptographic secret or public key. |
| MissingJWTVerification.rb:18:29:18:51 | token_without_signature | is not verified with a cryptographic secret or public key. |
